### PR TITLE
[FEM.HyperElastic] Implement buildStiffnessMatrix for StandardTetrahedralFEMForceField

### PR DESCRIPTION
**Pull request by alxbilger**
A task from #3967

______________________________________________________

By submitting this pull request, I acknowledge that  
**I have read, understand, and agree [SOFA Developer Certificate of Origin (DCO)](https://github.com/sofa-framework/sofa/blob/master/CONTRIBUTING.md#sofa-developer-certificate-of-origin-dco)**.
______________________________________________________

**Reviewers will merge this pull-request only if**  
- it builds with SUCCESS for all platforms on the CI.
- it does not generate new warnings.
- it does not generate new unit test failures.
- it does not generate new scene test failures.
- it does not break API compatibility.
- it is more than 1 week old (or has fast-merge label).

### CODE REVIEW

**Comment by alxbilger** [comment]
[ci-build][with-all-tests]

File: Sofa/Component/SolidMechanics/FEM/HyperElastic/src/sofa/component/solidmechanics/fem/hyperelastic/StandardTetrahedralFEMForceField.inl [hunk 1]
```diff
@@ -566,6 +566,35 @@ void  StandardTetrahedralFEMForceField<DataTypes>::addKToMatrix(sofa::linearalge
...
+
+    for (sofa::Size l = 0; l < nbEdges; ++l)
+    {
+        const auto& einfo = edgeInf[l];
+        const Index node0 = edgeArray[l][0];
+        const Index node1 = edgeArray[l][1];
+        const Index N0 = 3 * node0;
+        const Index N1 = 3 * node1;
+
+        const Matrix3& stiff = einfo.DfDx;
```
Comment (epernod) [inline Sofa/Component/SolidMechanics/FEM/HyperElastic/src/sofa/component/solidmechanics/fem/hyperelastic/StandardTetrahedralFEMForceField.inl:588]: Just comparing to AddKToMatrix, No need for the kFact ?
`const Matrix3 stiff = einfo.DfDx * (Real)kFact;`

and any idea why there is an offset in the other version?
`const Matrix3 stiff = einfo.DfDx * (Real)kFact;`

Comment (alxbilger) [reply Sofa/Component/SolidMechanics/FEM/HyperElastic/src/sofa/component/solidmechanics/fem/hyperelastic/StandardTetrahedralFEMForceField.inl:588]: Yes, one of the goal of the refactoring of the matrix assembly was to get rid of the offset and the k factor. You no longer need them in `buildStiffnessMatrix`. Instead, they are used in the matrix assembly component.

Comment (epernod) [reply Sofa/Component/SolidMechanics/FEM/HyperElastic/src/sofa/component/solidmechanics/fem/hyperelastic/StandardTetrahedralFEMForceField.inl:588]: ok sorry for the inattentive questions

**Pull request merged**